README.md

```diff
@@ -17,14 +17,25 @@ frac_year   | YYYY.fffff
 
 Important differences:
 
-- In CxoTime the date '2000:001' is '2000:001:00:00:00' instead of
-  '2000:001:12:00:00' in DateTime.  In most cases this interpretation is more
-  rational and expected.
-- In CxoTime the date '2001-01-01T00:00:00' is UTC, while in DateTime
-  that is interpreted as TT.
+- In `CxoTime` the date '2000:001' is '2000:001:00:00:00' instead of
+  '2000:001:12:00:00' in `DateTime` (prior to version 4.0).  In most cases this
+  interpretation is more rational and expected.
 
-The standard built-in Time formats that are available in CxoTime are:
+- In `CxoTime` the date '2001-01-01T00:00:00' is UTC by default, while in
+  `DateTime` that is interpreted as TT by default.  This is triggered by
```

Comment:
Fine with the doc but I don't remember having a conversation about these defaults.

Is it more confusing that there's a difference between Datetime and CxoTime in this regard (what is in there) or more confusing to have different default time systems depending on the string (what you'd have if you used the DateTime logic)?

Reply:
I think the `DateTime` behavior (different time system depending on there being a `T` or not) is very confusing for anyone but Arnold and people steeped in FITS. There might be some issue reading FITS times in CXC files, so that is a concern but only if we migrate core cron processing to CxoTime, which is not likely in the near future.